src/components/AmountFilter.jsx

```diff
@@ -8,7 +8,6 @@ import AmountTextField from './AmountTextField';
 
 const useStyles = makeStyles((theme) => ({
   buttonStyle: {
-    margin: theme.spacing(1),
```

Comment:
Removed this because it was causing a weird alignment issue in small screens:
![image](https://user-images.githubusercontent.com/8023233/134790632-7bdc7b05-5df3-4932-bd49-9d0586377494.png)